templates/default/rt/rtsearch.html

```diff
@@ -207,31 +194,35 @@ <H1>{$layout.pagetitle}</H1>
 		</TD>
 	</TR>
 	<TR>
-        <TD class="bold nobr"><img src="img/netdev.gif" alt=""> {trans("Device:")}</TD>
+        <TD class="bold nobr">{icon name="netdev" label="Device:" class="fa-fw"}</TD>
 		<TD class="nobr" id="rtnetdevs">
 			{include file="rt/rtnetdevs.html" form="search"}
 		</TD>
 	</TR>
-        <TR>
-		<TD class="bold nobr">{box_row icon="verifier" label="Verifier:" field_id="rtverifiers"}{/box_row}</TD>
+    <TR>
+		<TD class="bold nobr">
+			{icon name="verifier" label="Verifier:" field_id="rtverifiers"}
+		</TD>
 		<TD>
-                        <SELECT size="1" name="search[verifierid]" {tip text="Select user/owner of the ticket"}>
-                                <OPTION value="0">{trans("- all owners -")}</OPTION>
-                                {foreach $userlist as $item}
-                                <OPTION value="{$item.id}" {if $item.id == $search.verifierid}SELECTED{/if}>{$item.rname}</OPTION>
-                                {/foreach}
-                        </SELECT>
-		</TD>
-        </TR>
-        <TR>
-		<TD class="bold nobr"><img src="img/calendar.gif" alt=""> {trans("Deadline expired:")}</TD>
+			<SELECT size="1" name="search[verifierid]" {tip text="Select user/owner of the ticket"}>
+				<OPTION value="0">{trans("- all owners -")}</OPTION>
+				{foreach $userlist as $item}
+				<OPTION value="{$item.id}" {if $item.id == $search.verifierid}SELECTED{/if}>{$item.rname}</OPTION>
+				{/foreach}
+			</SELECT>
+		</TD>
+    </TR>
+    <TR>
+		<TD class="bold nobr">
+			{icon name="deadline" label="Deadline expired:" class="fa-fw"}
+		</TD>
```

Comment:
A tu co taki duży rewrite? Potem problem z przenoszeniem zmian między gałęziami.

Reply:
Nie no to jest jasne, że to "MASTER-ONLY" tutaj poszedł bardzo duży rewrite. Dlaczego? 

Starałem się uporządkować kwestie związane z fontawesome w module RT - przejrzałem formularz po formularzu i to co moje oczy wypatrzyły to pozmieniałem np.
1. Włożyłem etykietki do {icon co wymagało usunięcia części komórek co z kolei przełożyło się na formatowanie pozostałych rzeczy i dodania klas bold w TD, żeby zachować pierwotny wygląd.
2. Powymieniałem prawie wszystkie <i class="lms-ui-icon-nazwa"> na {icon.
3. W kilku miejscach poprawiłem formatowanie bo było wystrzelone w kosmos.
4. W kilku miejscach dorzuciłem klasy "fa-fw".
5. Dorobiłem obsługę parametru label_style w funkcji Smarty (oddzielny commit tak, żeby było można wrzucić do stable).

Reply:
Zakumałem o co chodzi z tym formatowaniem w PHPStorm. Jest opcja detekcji wcięć w pliku  jeśli się ją włączy (domyślnie włączona) to on olewa ustawienia space per tabs. 

Z drugiej strony webeditor gita pokazuje inaczej 4 spacje i tabulacje 
(i do tego inaczej pokazuje w oknie kiedy oglądamy plik a inaczej gdy go edytujemy)